tracker_audit.go

```
@@ -68,7 +79,14 @@ type RunSummary struct {
 // and activity.jsonl directly under it. For user-supplied input, resolve
 // the path via ResolveRunDir or use MostRecentRunID first, which enforce
 // the .tracker/runs/<runID> layout.
-func Audit(runDir string) (*AuditReport, error) {
+//
+// ctx is accepted for future extensibility. A nil context is treated as
+// context.Background().
+func Audit(ctx context.Context, runDir string, opts ...AuditConfig) (*AuditReport, error) {
+	if ctx == nil {
+		ctx = context.Background()
```

golangci-lint (2.11.4)

[error] ineffectual assignment to ctx

(ineffassign)